video/out/opengl/context_angle.c

```diff
@@ -662,10 +665,20 @@ static int angle_init(struct MPGLContext *ctx, int flags)
 
     // Create the underlying EGL surface implementation
     bool surface_ok = false;
-    if ((!surface_ok && o->egl_windowing == -1) || o->egl_windowing == 0)
+    if ((!surface_ok && o->egl_windowing == -1) || o->egl_windowing == 0) {
         surface_ok = d3d11_swapchain_surface_create(ctx, flags);
-    if ((!surface_ok && o->egl_windowing == -1) || o->egl_windowing == 1)
+        if (surface_ok) {
+            if (p->dxgi_swapchain1)
+                MP_VERBOSE(vo, "Using DXGI 1.2+\n");
+            else
+                MP_VERBOSE(vo, "Using DXGI 1.1\n");
```

Comment:
Hmm, because it has an else block or because it's nested inside another if block?

contribute.md says "if the body of an if statement uses braces, the else branch should also use braces," but I took to mean that if neither use braces, it's okay.